home.html

```diff
@@ -17,7 +18,7 @@
 	  </header>
 	  <main>
 	  	<div class="homebackground">
-		  	<div class="homequote">All those moments will be lost in time, like tears in the rain.
```

Comment:
My homequote is currently coming up as "all those moments..." rather than your new quote. Not sure why that is.